src/Plugins/RpcServer/RpcServer.Blockchain.cs

```diff
@@ -84,8 +113,16 @@ protected virtual JToken GetBlockHash(JArray _params)
             throw new RpcException(RpcError.UnknownHeight);
         }
 
+        /// <summary>
+        /// Gets a block header by its hash or index.
+        /// </summary>
+        /// <param name="_params">
+        /// An array containing the block header hash or index as the first element,
+        /// and an optional boolean indicating whether to return verbose information.
+        /// </param>
+        /// <returns>The block header data as a <see cref="JToken"/>. In json format if the second item of _params is true, otherwise base64string.</returns>
```

Comment:
```suggestion
        /// <returns>The block header data as a <see cref="JToken"/>. In json format if the second item of _params is true, otherwise Base64-encoded byte array.</returns>
```